### ISSUE TITLE
MMEX blocked by macOS Gatekeeper quarantine

### ISSUE THREAD

**Issue by wirmike** [moneymanagerex/moneymanagerex#1501]
Your OS:
- [ ] Windows
- [x] Mac OSX (High Sierra 10.13.3)
- [ ] Linux 
- [ ] Android -> please report bugs to moneymanagerex/android-money-manager-ex repo!

MMEX version:
- [ ] 1.4.x
- [x] 1.3.x (1.3.3)
- [ ] 1.2.x
- [ ] 1.1 or older
- [ ] development version (git master branch)

This report is probably a mix of two issues.  The original re-occurence
of report #1195, and principally the issue of being unable to reinstall.
The narrative follows.  Unfortunately, events have passed that would have enabled a screen-shot/s to explain. 
The narrative ...
Re the bug/issue I reported on #1195: the ‘bug/issue’ has re-appeared and has proved impossible to correct!
It has to be said that since loading MMEX v1.3.3 everything has been (hitherto) good.
What seems to have happened was that I caught my MacMouse on the edge of my MacBookPro (the top edge at the opposite end to the mouse’s Apple logo.
At that point the Navigation panel ‘flipped’ from the LHS of the screen to the top - between the toolbar and the open accounts file.  Nothing that I did seemed to restore the Panel to its ‘rightful’ position on the left!
It seems that an erroneous 'click/s' on the mouse may have triggered something with the pointer at whatever place it was on the screen - maybe - I may never know, so fleeting a moment it was.
So, I tried to reinstall the App.  Big mistake!
When I tried to launch the result of a reinstall the app reported ““mmex.app” is damaged and can’t be opened.”
No matter how many times I downloaded and retried installing, restarted the machine and ‘trashed’ the newly created apps, they all resulted in an unusable MMEX app.
I then remembered that I had some similar issue when I tried reinstalling - resulting in the error report #1195.

At present I cannot use my MMEX from the Mac and really need some guidance.

HOWEVER, MMEX in the Win7 parallels side of the Mac does work and seems to be able to access the shared files.  I have started using this (now sole route) - I use MMEX REALLY HEAVILY!!!.

I actually FAR prefer the Mac version of MMEX - the Windows one is nice and fine - but for looks and general usability OSX has it.

My Mac is at High Sierra 10.13.3.

I would be grateful for any help.

Regards,

**Comment by bestlem** [comment]
For the crash. Try starting mmex from the terminal.

run 


/Applications/mmex.app/Contents/MacOS/mmex

What is printed there?

**Comment by wirmike** [comment]
Bestle,

Does this help?

Many thanks for your rapid answer.

This includes actually running the app and checking out a file.
++++++++++++++++++++++++++++++++++++++++++++++

Last login: Sun Mar 25 06:19:26 on ttys000
MGVs-MacBook-Pro:~ mikedavies$ /Applications/mmex.app/Contents/MacOS/mmex
http://www.google-analytics.com/collect?an=MoneyManagerEx&av=1.3.3&cd1=mac&cid=mac_20170220154938407&dp=/frame/frame&dt=Navigation&sd=32-bits&sr=1440x900&t=pageview&tid=UA-51521761-6&ul=english-uk&v=1 
User-Agent: Mac OS X (Darwin 17.4.0 x86_64)

timeout
http://www.google-analytics.com/collect?an=MoneyManagerEx&av=1.3.3&cd1=mac&cid=mac_20170220154938407&dp=/frame/dialog&dt=Money%20Manager%20Ex%20-%20Version:%201.3.3&sd=32-bits&sr=1440x900&t=pageview&tid=UA-51521761-6&ul=english-uk&v=1 
User-Agent: Mac OS X (Darwin 17.4.0 x86_64)

timeout
http://www.google-analytics.com/collect?an=MoneyManagerEx&av=1.3.3&cd1=mac&cid=mac_20170220154938407&dp=/frame/dialog&dt=Money%20Manager%20Ex%20-%20Version:%201.3.3&sd=32-bits&sr=1440x900&t=pageview&tid=UA-51521761-6&ul=english-uk&v=1 
User-Agent: Mac OS X (Darwin 17.4.0 x86_64)

http://www.google-analytics.com/collect?an=MoneyManagerEx&av=1.3.3&cd1=mac&cid=mac_20170220154938407&dp=/frame/panel/mmHomePagePanel&dt=mmHomePagePanel&sd=32-bits&sr=1440x900&t=pageview&tid=UA-51521761-6&ul=english-uk&v=1 
User-Agent: Mac OS X (Darwin 17.4.0 x86_64)

timeout
timeout
objc[60706]: Class FIFinderSyncExtensionHost is implemented in both /System/Library/PrivateFrameworks/FinderKit.framework/Versions/A/FinderKit (0x7fff8fe68b68) and /System/Library/PrivateFrameworks/FileProvider.framework/OverrideBundles/FinderSyncCollaborationFileProviderOverride.bundle/Contents/MacOS/FinderSyncCollaborationFileProviderOverride (0x1197f6cd8). One of the two will be used. Which one is undefined.
http://www.google-analytics.com/collect?an=MoneyManagerEx&av=1.3.3&cd1=mac&cid=mac_20170220154938407&dp=/frame/panel/mmCheckingPanel&dt=mmCheckingPanel&sd=32-bits&sr=1440x900&t=pageview&tid=UA-51521761-6&ul=english-uk&v=1 
User-Agent: Mac OS X (Darwin 17.4.0 x86_64)

timeout
MGVs-MacBook-Pro:~ mikedavies$

++++++++++++++++++++++++++++++++++++++++++++++

Regards,

MikeD

**Comment by vomikan** [comment]
Possible solution is to delete mmexini.db3 file.

**Comment by wirmike** [comment]
Nikolay,

Thank you for the reply.

Just got in from Liverpool.  My daughter was running in the annual BTR 1/2 Marathon.

I tried to find the mmexini.db3 file in Finder, but it does not seem to exist!
There may well be something of that ilk in the Mac app, but I would not know, if it is possible, to get at it.

Unless there is something that v.1.3.3 does not like in the current version of Mac OS whilst installing, then I am at present somewhat mystified.

Stefano actually suggested similar on 24JNE last year, re the ‘.db3’ file and I was unable to find any then.
It would seem that being a Mac OS App, then there is something else going on, maybe.
From what I remember, the issue seemed to cleared up by upgrading from v.1.3.1 to v.1.3.3 - but the Mac then was at an earlier version of Sierra.

I have appended an extract from the Finder screen showing the empty search result for the more general ‘.db3’ files.

Hope that helps.

Regards,

MkeD

Appendix:

**Comment by bestlem** [comment]
Try ~/Library/Application Support/MoneyManagerEx/mmexini.db3 Unfortunately Apple has decided to hode ~/Library from the default use of Finder and searches.

**Comment by wirmike** [comment]
Bestlem,

Ah!  I'll give your idea a whirl!

I suppose it's like 'hiding' 'system' files in Windows.

I'll let you know how I go.

Regards,

Mike

**Comment by wirmike** [comment]
Bestlem,

I tried as you suggested.

I discovered another aspect of the Mac - I needed to go via the “Go” menu at the top of the Mac Desktop.

Actually the path was:

~/Library/Application Support/MMEX/mmexini.db3
>

**Comment by wirmike** [comment]
Bestlem,

I tried as you suggested.

In trying to find the hidden files I discovered another aspect of the Mac - I needed to go via the “Go” menu at the top of the Finder screen.

Actually, I discovered that the path was actually:

~/Library/Application Support/mmex/mmexini.db3.

I sent 'mmexini.db3’ to Trash.

I then retried the MMEX app, but alas, it still brought up the same ‘damaged can’t be opened' error.

To keep things ’straight’ I restored the file from Trash.

Hmmm!

Regards,

Mike

**Comment by wirmike** [comment]
Bestle,

Done.

Does this help?
==========================================================
Last login: Sun Mar 25 20:21:22 on console
MGVs-MacBook-Pro:~ mikedavies$ ls -lR /Applications/MMEX.app
total 0
drwxr-xr-x@ 6 mikedavies  admin  192  7 Oct 17:55 Contents

/Applications/MMEX.app/Contents:
total 8
-rw-r--r--@  1 mikedavies  admin  1607  7 Oct 18:07 Info.plist
drwxr-xr-x@  3 mikedavies  admin    96  7 Oct 17:55 MacOS
drwxr-xr-x@  4 mikedavies  admin   128  7 Oct 17:55 Resources
drwxr-xr-x@ 10 mikedavies  admin   320  7 Oct 17:55 SharedSupport

/Applications/MMEX.app/Contents/MacOS:
total 33536
-rwxr-xr-x@ 1 mikedavies  admin  17167240  7 Oct 18:07 mmex

/Applications/MMEX.app/Contents/Resources:
total 352
-rw-r--r--@ 1 mikedavies  admin   47300  7 Oct 18:07 mmdb.icns
-rw-r--r--@ 1 mikedavies  admin  130000  7 Oct 18:07 mmex.icns

/Applications/MMEX.app/Contents/SharedSupport:
total 120
drwxr-xr-x@  3 mikedavies  admin     96  7 Oct 17:55 applications
-rw-r--r--@  1 mikedavies  admin   9614  7 Oct 18:07 contrib.txt
drwxr-xr-x@  3 mikedavies  admin     96  7 Oct 17:55 doc
drwxr-xr-x@  3 mikedavies  admin     96  7 Oct 17:55 icons
-rw-r--r--@  1 mikedavies  admin   3468  7 Oct 18:07 license.txt
drwxr-xr-x@ 41 mikedavies  admin   1312  7 Oct 18:07 po
drwxr-xr-x@  8 mikedavies  admin    256  7 Oct 17:55 res
-rw-r--r--@  1 mikedavies  admin  42913  7 Oct 18:07 version.txt

/Applications/MMEX.app/Contents/SharedSupport/applications:
total 8
-rw-r--r--@ 1 mikedavies  admin  2608  7 Oct 18:07 mmex.desktop

/Applications/MMEX.app/Contents/SharedSupport/doc:
total 0
drwxr-xr-x@ 6 mikedavies  admin  192  7 Oct 17:55 mmex
==========================================================

Regards,

Mike

**Comment by bestlem** [comment]
That is not complete - where are you downloading from ?

**Comment by wirmike** [comment]
Bestlem,

Oops!

Hadn’t noticed that my $ prompt at the end was missing!

Hope this is now complete.
 
============================================================
Last login: Tue Mar 27 15:45:33 on ttys000
MGVs-MacBook-Pro:~ mikedavies$ ls -lR /Applications/MMEX.app
total 0
drwxr-xr-x@ 6 mikedavies  admin  192  7 Oct 17:55 Contents

/Applications/MMEX.app/Contents:
total 8
-rw-r--r--@  1 mikedavies  admin  1607  7 Oct 18:07 Info.plist
drwxr-xr-x@  3 mikedavies  admin    96  7 Oct 17:55 MacOS
drwxr-xr-x@  4 mikedavies  admin   128  7 Oct 17:55 Resources
drwxr-xr-x@ 10 mikedavies  admin   320  7 Oct 17:55 SharedSupport

/Applications/MMEX.app/Contents/MacOS:
total 33536
-rwxr-xr-x@ 1 mikedavies  admin  17167240  7 Oct 18:07 mmex

/Applications/MMEX.app/Contents/Resources:
total 352
-rw-r--r--@ 1 mikedavies  admin   47300  7 Oct 18:07 mmdb.icns
-rw-r--r--@ 1 mikedavies  admin  130000  7 Oct 18:07 mmex.icns

/Applications/MMEX.app/Contents/SharedSupport:
total 120
drwxr-xr-x@  3 mikedavies  admin     96  7 Oct 17:55 applications
-rw-r--r--@  1 mikedavies  admin   9614  7 Oct 18:07 contrib.txt
drwxr-xr-x@  3 mikedavies  admin     96  7 Oct 17:55 doc
drwxr-xr-x@  3 mikedavies  admin     96  7 Oct 17:55 icons
-rw-r--r--@  1 mikedavies  admin   3468  7 Oct 18:07 license.txt
drwxr-xr-x@ 41 mikedavies  admin   1312  7 Oct 18:07 po
drwxr-xr-x@  8 mikedavies  admin    256  7 Oct 17:55 res
-rw-r--r--@  1 mikedavies  admin  42913  7 Oct 18:07 version.txt

/Applications/MMEX.app/Contents/SharedSupport/applications:
total 8
-rw-r--r--@ 1 mikedavies  admin  2608  7 Oct 18:07 mmex.desktop

/Applications/MMEX.app/Contents/SharedSupport/doc:
total 0
drwxr-xr-x@ 6 mikedavies  admin  192  7 Oct 17:55 mmex

/Applications/MMEX.app/Contents/SharedSupport/doc/mmex:
total 120
-rw-r--r--@  1 mikedavies  admin   9614  7 Oct 18:07 contrib.txt
drwxr-xr-x@ 27 mikedavies  admin    864  7 Oct 17:55 help
-rw-r--r--@  1 mikedavies  admin   3468  7 Oct 18:07 license.txt
-rw-r--r--@  1 mikedavies  admin  42913  7 Oct 18:07 version.txt

/Applications/MMEX.app/Contents/SharedSupport/doc/mmex/help:
total 2240
-rw-r--r--@  1 mikedavies  admin  247169  7 Oct 18:07 GRM.gif
-rw-r--r--@  1 mikedavies  admin    3693  7 Oct 18:07 backtotop.png
-rw-r--r--@  1 mikedavies  admin    3592  7 Oct 18:07 btn_donateCC_LG.gif
-rw-r--r--@  1 mikedavies  admin    2565  7 Oct 18:07 budget.html
-rw-r--r--@  1 mikedavies  admin   13884  7 Oct 18:07 budget_editor.png
-rw-r--r--@  1 mikedavies  admin   17894  7 Oct 18:07 budget_entry_details.png
-rw-r--r--@  1 mikedavies  admin   37213  7 Oct 18:07 budget_grid.png
-rw-r--r--@  1 mikedavies  admin    1083  7 Oct 18:07 contents.js
drwxr-xr-x@ 10 mikedavies  admin     320  7 Oct 17:55 french
-rw-r--r--@  1 mikedavies  admin    1462  7 Oct 18:07 general_report_manager.html
drwxr-xr-x@  6 mikedavies  admin     192  7 Oct 17:55 hungarian
-rw-r--r--@  1 mikedavies  admin   88216  7 Oct 18:07 index.html
drwxr-xr-x@  8 mikedavies  admin     256  7 Oct 17:55 italian
-rw-r--r--@  1 mikedavies  admin   14831  7 Oct 18:07 master.css
-rw-r--r--@  1 mikedavies  admin   70461  7 Oct 18:07 mmex_concept.png
-rw-r--r--@  1 mikedavies  admin  187142  7 Oct 18:07 mmex_db_structure.png
drwxr-xr-x@ 16 mikedavies  admin     512  7 Oct 17:55 polish
drwxr-xr-x@  6 mikedavies  admin     192  7 Oct 17:55 russian
drwxr-xr-x@  5 mikedavies  admin     160  7 Oct 17:55 spanish
-rw-r--r--@  1 mikedavies  admin   63522  7 Oct 18:07 stock_shares_new_shares.png
-rw-r--r--@  1 mikedavies  admin   77190  7 Oct 18:07 stock_shares_new_stock.png
-rw-r--r--@  1 mikedavies  admin  116195  7 Oct 18:07 stock_shares_portfolio.png
-rw-r--r--@  1 mikedavies  admin  119409  7 Oct 18:07 stock_shares_share_account.png
-rw-r--r--@  1 mikedavies  admin   27097  7 Oct 18:07 stock_shares_view_transactions.png
-rw-r--r--@  1 mikedavies  admin   13174  7 Oct 18:07 stocks_and_shares.html

/Applications/MMEX.app/Contents/SharedSupport/doc/mmex/help/french:
total 408
-rw-r--r--@ 1 mikedavies  admin  15262  7 Oct 18:07 account.jpg
-rw-r--r--@ 1 mikedavies  admin  21112  7 Oct 18:07 add_acc_wizard1.jpg
-rw-r--r--@ 1 mikedavies  admin  20174  7 Oct 18:07 add_acc_wizard2.jpg
-rw-r--r--@ 1 mikedavies  admin  31526  7 Oct 18:07 add_acc_wizard3.jpg
-rw-r--r--@ 1 mikedavies  admin  53955  7 Oct 18:07 index.html
-rw-r--r--@ 1 mikedavies  admin   6425  7 Oct 18:07 investment.html
-rw-r--r--@ 1 mikedavies  admin  24595  7 Oct 18:07 main_screen.jpg
-rw-r--r--@ 1 mikedavies  admin  16988  7 Oct 18:07 mmex_concept.png

/Applications/MMEX.app/Contents/SharedSupport/doc/mmex/help/hungarian:
total 344
-rw-r--r--@ 1 mikedavies  admin   3351  7 Oct 18:07 budget.html
-rw-r--r--@ 1 mikedavies  admin  90679  7 Oct 18:07 index.html
-rw-r--r--@ 1 mikedavies  admin   8411  7 Oct 18:07 investment.html
-rw-r--r--@ 1 mikedavies  admin  64047  7 Oct 18:07 mmex_concept_hu.png

/Applications/MMEX.app/Contents/SharedSupport/doc/mmex/help/italian:
total 800
-rw-r--r--@ 1 mikedavies  admin  122648  7 Oct 18:07 index.html
-rw-r--r--@ 1 mikedavies  admin    7969  7 Oct 18:07 investment.html
-rw-r--r--@ 1 mikedavies  admin   58122  7 Oct 18:07 mmex_concept.png
-rw-r--r--@ 1 mikedavies  admin   50685  7 Oct 18:07 stock_dialog.jpg
-rw-r--r--@ 1 mikedavies  admin   31514  7 Oct 18:07 stock_options.jpg
-rw-r--r--@ 1 mikedavies  admin  131007  7 Oct 18:07 stock_view.jpg

/Applications/MMEX.app/Contents/SharedSupport/doc/mmex/help/polish:
total 1592
-rw-r--r--@ 1 mikedavies  admin  247169  7 Oct 18:07 GRM.gif
-rw-r--r--@ 1 mikedavies  admin    2428  7 Oct 18:07 btn_donateCC_LG.gif
-rw-r--r--@ 1 mikedavies  admin    2869  7 Oct 18:07 budget.html
-rw-r--r--@ 1 mikedavies  admin   18105  7 Oct 18:07 budget_editor.png
-rw-r--r--@ 1 mikedavies  admin   13745  7 Oct 18:07 budget_entry_details.png
-rw-r--r--@ 1 mikedavies  admin   18732  7 Oct 18:07 budget_grid.png
-rw-r--r--@ 1 mikedavies  admin    3590  7 Oct 18:07 general_report_manager.html
-rw-r--r--@ 1 mikedavies  admin   81779  7 Oct 18:07 index.html
-rw-r--r--@ 1 mikedavies  admin    5302  7 Oct 18:07 investment.html
-rw-r--r--@ 1 mikedavies  admin   71831  7 Oct 18:07 mmex_concept.png
-rw-r--r--@ 1 mikedavies  admin  209891  7 Oct 18:07 mmex_structure.jpg
-rw-r--r--@ 1 mikedavies  admin   23069  7 Oct 18:07 stock_dialog.jpg
-rw-r--r--@ 1 mikedavies  admin   14870  7 Oct 18:07 stock_options.jpg
-rw-r--r--@ 1 mikedavies  admin   76650  7 Oct 18:07 stock_view.jpg

/Applications/MMEX.app/Contents/SharedSupport/doc/mmex/help/russian:
total 376
-rw-r--r--@ 1 mikedavies  admin  70769  7 Oct 18:07 index.html
-rw-r--r--@ 1 mikedavies  admin   4171  7 Oct 18:07 investment.html
-rw-r--r--@ 1 mikedavies  admin  70046  7 Oct 18:07 mmex_concept.png
-rw-r--r--@ 1 mikedavies  admin  33198  7 Oct 18:07 stock_dialog.png

/Applications/MMEX.app/Contents/SharedSupport/doc/mmex/help/spanish:
total 200
-rw-r--r--@ 1 mikedavies  admin   2838  7 Oct 18:07 budget.html
-rw-r--r--@ 1 mikedavies  admin  86809  7 Oct 18:07 index.html
-rw-r--r--@ 1 mikedavies  admin   6768  7 Oct 18:07 investment.html

/Applications/MMEX.app/Contents/SharedSupport/icons:
total 0
drwxr-xr-x@ 3 mikedavies  admin  96  7 Oct 17:55 hicolor

/Applications/MMEX.app/Contents/SharedSupport/icons/hicolor:
total 0
drwxr-xr-x@ 3 mikedavies  admin  96  7 Oct 17:55 scalable

/Applications/MMEX.app/Contents/SharedSupport/icons/hicolor/scalable:
total 0
drwxr-xr-x@ 3 mikedavies  admin  96  7 Oct 17:55 apps

/Applications/MMEX.app/Contents/SharedSupport/icons/hicolor/scalable/apps:
total 24
-rw-r--r--@ 1 mikedavies  admin  8577  7 Oct 18:07 mmex.svg

/Applications/MMEX.app/Contents/SharedSupport/po:
total 4800
-rw-r--r--@  1 mikedavies  admin   28270  7 Oct 17:59 arabic.mo
-rw-r--r--@  1 mikedavies  admin  103003  7 Oct 17:59 bulgarian.mo
-rw-r--r--@  1 mikedavies  admin   32056  7 Oct 17:59 chinese_chs.mo
-rw-r--r--@  1 mikedavies  admin   22004  7 Oct 17:59 chinese_zh.mo
-rw-r--r--@  1 mikedavies  admin   24524  7 Oct 17:59 croatian.mo
-rw-r--r--@  1 mikedavies  admin   80646  7 Oct 17:59 czech.mo
-rw-r--r--@  1 mikedavies  admin  135499  7 Oct 17:59 dutch.mo
-rw-r--r--@  1 mikedavies  admin   52740  7 Oct 17:59 dutch_be.mo
drwxr-xr-x@ 40 mikedavies  admin    1280  7 Oct 18:07 en
-rw-r--r--@  1 mikedavies  admin     822  7 Oct 17:59 english-uk.mo
-rw-r--r--@  1 mikedavies  admin     558  7 Oct 17:59 english.mo
-rw-r--r--@  1 mikedavies  admin   84990  7 Oct 17:59 french.mo
-rw-r--r--@  1 mikedavies  admin   81010  7 Oct 17:59 german.mo
-rw-r--r--@  1 mikedavies  admin   74641  7 Oct 17:59 greek.mo
-rw-r--r--@  1 mikedavies  admin  149648  7 Oct 17:59 hebrew.mo
-rw-r--r--@  1 mikedavies  admin    3334  7 Oct 17:59 hindi.mo
-rw-r--r--@  1 mikedavies  admin  139899  7 Oct 17:59 hungarian.mo
-rw-r--r--@  1 mikedavies  admin   40314  7 Oct 17:59 indonesian.mo
-rw-r--r--@  1 mikedavies  admin  135440  7 Oct 17:59 italian.mo
-rw-r--r--@  1 mikedavies  admin   88874  7 Oct 17:59 japanese.mo
-rw-r--r--@  1 mikedavies  admin   27741  7 Oct 17:59 latvian.mo
-rw-r--r--@  1 mikedavies  admin   92164  7 Oct 17:59 lithuanian.mo
-rw-r--r--@  1 mikedavies  admin   18095  7 Oct 17:59 norwegian.mo
-rw-r--r--@  1 mikedavies  admin  133364  7 Oct 17:59 polish.mo
-rw-r--r--@  1 mikedavies  admin   83023  7 Oct 17:59 portuguese.mo
-rw-r--r--@  1 mikedavies  admin   80775  7 Oct 17:59 portuguese_br.mo
-rw-r--r--@  1 mikedavies  admin   24286  7 Oct 17:59 portuguese_portugal.mo
-rw-r--r--@  1 mikedavies  admin   27598  7 Oct 17:59 romanian.mo
-rw-r--r--@  1 mikedavies  admin  151672  7 Oct 17:59 russian.mo
-rw-r--r--@  1 mikedavies  admin   33171  7 Oct 17:59 serbian.mo
-rw-r--r--@  1 mikedavies  admin   23837  7 Oct 17:59 serbo-croatian.mo
-rw-r--r--@  1 mikedavies  admin   82225  7 Oct 17:59 slovak.mo
-rw-r--r--@  1 mikedavies  admin   28075  7 Oct 17:59 slovenian.mo
-rw-r--r--@  1 mikedavies  admin   84397  7 Oct 17:59 spanish.mo
-rw-r--r--@  1 mikedavies  admin   19879  7 Oct 17:59 swedish.mo
-rw-r--r--@  1 mikedavies  admin   35161  7 Oct 17:59 tamil.mo
-rw-r--r--@  1 mikedavies  admin   91205  7 Oct 17:59 turkish.mo
-rw-r--r--@  1 mikedavies  admin   28565  7 Oct 17:59 ukrainian.mo
-rw-r--r--@  1 mikedavies  admin   44017  7 Oct 17:59 vietnamese.mo

/Applications/MMEX.app/Contents/SharedSupport/po/en:
total 4800
-rw-r--r--@ 1 mikedavies  admin   28270  7 Oct 18:07 arabic.mo
-rw-r--r--@ 1 mikedavies  admin  103003  7 Oct 18:07 bulgarian.mo
-rw-r--r--@ 1 mikedavies  admin   32056  7 Oct 18:07 chinese_chs.mo
-rw-r--r--@ 1 mikedavies  admin   22004  7 Oct 18:07 chinese_zh.mo
-rw-r--r--@ 1 mikedavies  admin   24524  7 Oct 18:07 croatian.mo
-rw-r--r--@ 1 mikedavies  admin   80646  7 Oct 18:07 czech.mo
-rw-r--r--@ 1 mikedavies  admin  135499  7 Oct 18:07 dutch.mo
-rw-r--r--@ 1 mikedavies  admin   52740  7 Oct 18:07 dutch_be.mo
-rw-r--r--@ 1 mikedavies  admin     822  7 Oct 18:07 english-uk.mo
-rw-r--r--@ 1 mikedavies  admin     558  7 Oct 18:07 english.mo
-rw-r--r--@ 1 mikedavies  admin   84990  7 Oct 18:07 french.mo
-rw-r--r--@ 1 mikedavies  admin   81010  7 Oct 18:07 german.mo
-rw-r--r--@ 1 mikedavies  admin   74641  7 Oct 18:07 greek.mo
-rw-r--r--@ 1 mikedavies  admin  149648  7 Oct 18:07 hebrew.mo
-rw-r--r--@ 1 mikedavies  admin    3334  7 Oct 18:07 hindi.mo
-rw-r--r--@ 1 mikedavies  admin  139899  7 Oct 18:07 hungarian.mo
-rw-r--r--@ 1 mikedavies  admin   40314  7 Oct 18:07 indonesian.mo
-rw-r--r--@ 1 mikedavies  admin  135440  7 Oct 18:07 italian.mo
-rw-r--r--@ 1 mikedavies  admin   88874  7 Oct 18:07 japanese.mo
-rw-r--r--@ 1 mikedavies  admin   27741  7 Oct 18:07 latvian.mo
-rw-r--r--@ 1 mikedavies  admin   92164  7 Oct 18:07 lithuanian.mo
-rw-r--r--@ 1 mikedavies  admin   18095  7 Oct 18:07 norwegian.mo
-rw-r--r--@ 1 mikedavies  admin  133364  7 Oct 18:07 polish.mo
-rw-r--r--@ 1 mikedavies  admin   83023  7 Oct 18:07 portuguese.mo
-rw-r--r--@ 1 mikedavies  admin   80775  7 Oct 18:07 portuguese_br.mo
-rw-r--r--@ 1 mikedavies  admin   24286  7 Oct 18:07 portuguese_portugal.mo
-rw-r--r--@ 1 mikedavies  admin   27598  7 Oct 18:07 romanian.mo
-rw-r--r--@ 1 mikedavies  admin  151672  7 Oct 18:07 russian.mo
-rw-r--r--@ 1 mikedavies  admin   33171  7 Oct 18:07 serbian.mo
-rw-r--r--@ 1 mikedavies  admin   23837  7 Oct 18:07 serbo-croatian.mo
-rw-r--r--@ 1 mikedavies  admin   82225  7 Oct 18:07 slovak.mo
-rw-r--r--@ 1 mikedavies  admin   28075  7 Oct 18:07 slovenian.mo
-rw-r--r--@ 1 mikedavies  admin   84397  7 Oct 18:07 spanish.mo
-rw-r--r--@ 1 mikedavies  admin   19879  7 Oct 18:07 swedish.mo
-rw-r--r--@ 1 mikedavies  admin   35161  7 Oct 18:07 tamil.mo
-rw-r--r--@ 1 mikedavies  admin   91205  7 Oct 18:07 turkish.mo
-rw-r--r--@ 1 mikedavies  admin   28565  7 Oct 18:07 ukrainian.mo
-rw-r--r--@ 1 mikedavies  admin   44017  7 Oct 18:07 vietnamese.mo

/Applications/MMEX.app/Contents/SharedSupport/res:
total 1344
-rw-r--r--@ 1 mikedavies  admin  467807  7 Oct 18:07 ChartNew.js
-rw-r--r--@ 1 mikedavies  admin   10581  7 Oct 18:07 home_page.htt
-rw-r--r--@ 1 mikedavies  admin    8196  7 Oct 18:07 kaching.wav
-rw-r--r--@ 1 mikedavies  admin   14831  7 Oct 18:07 master.css
-rw-r--r--@ 1 mikedavies  admin  153607  7 Oct 18:07 mmex.ico
-rw-r--r--@ 1 mikedavies  admin   16628  7 Oct 18:07 sorttable.js
MGVs-MacBook-Pro:~ mikedavies$ 
============================================================

Regards,

Mike

**Comment by bestlem** [comment]
This is possibly the quarantine flag - Apple stops downloaded Apps from running.  See https://apple.stackexchange.com/a/253943/237 for some suggestions.

mmex.app would need to be signed to aloow users to use this directly without the command line.

**Comment by wirmike** [comment]
Bestlem,

I ran the command line as per below with the following result.
=========================================================
Last login: Tue Mar 27 20:59:20 on ttys000
MGVs-MacBook-Pro:~ mikedavies$  xattr /Applications/mmex.app
com.apple.quarantine
MGVs-MacBook-Pro:~ mikedavies$ 
=========================================================

Does this tell me that the app is okay?

From memory, on some earlier updates to MMEX, I have been asked if it was okay to run the app (updated one) from the internet and asked to confirm w/my admin id/pw.
Unless something untoward has happened to the app,  I would have no reason to doubt it.

Has OSX changed how it handles third-party apps,  or is this something different?  I follow your comment re the signing of the app.

Regards,

Mike

**Comment by bestlem** [comment]
That xattr result means that the app is in quarantine and so explains the not running issue.

You need to remove the quarantine flag

> sudo xattr -r -d com.apple.quarantine /Applications/mmex.app

This flag was created 3 or 4 OSX versions ago. Until Sierra there was a System Preference that would allow you to run unsigned apps - that has now gone so you get this having to remove the flag manually.

If this works for you I will raise another issue here stating more concisely the issue and what needs to be done - which might only be update the documentation,

**Comment by wirmike** [comment]
Bestlem,

======================================================================================
Last login: Tue Mar 27 20:59:20 on ttys000
MGVs-MacBook-Pro:~ mikedavies$  xattr /Applications/mmex.app
com.apple.quarantine
MGVs-MacBook-Pro:~ mikedavies$ sudo xattr -r -d com.apple.quarantine /Applications/mmex.app
Password:
mikedavies is not in the sudoers file.  This incident will be reported.
MGVs-MacBook-Pro:~ mikedavies$ 
======================================================================================

It seems that I will need to go into my Admin to add my UserID to the ’sudoers file’ - not that I’ve done THAT before - a veritable voyage of discovery!

I’ll report back.

Why can’t life be simple? (Rhetaoricle)

Regards,

Mke

**Comment by wirmike** [comment]
Bestlem,

Life is so simple that I can’t spell Rhetorical!

Mike

**Comment by slodki** [comment]
1. Instructions for users with quarantined app - whatever the reason was. Is there any GUI way to unblock app?
2. How to protect MMEX from being quarantined? Is signing app the only way?

**Comment by bestlem** [comment]
slodki, I think but not certain that 
1) See the other answers to the StackOverflow question https://apple.stackexchange.com/q/243687/237 and even then Apple have heavily blocked unsigned apps now, I don't think there is a GUI way of doing this.

2) Signing is the only way and all that will allow is allowing a GUI way of allowing a clean install. (although one of the answers implies it can be done by right clicking when opening but I am not certain of that user's settings) - Unfortunately I have an odd set up and don't have a plain user machine. Also see Apple docs https://help.apple.com/xcode/mac/current/#/dev033e997ca

Also note that the original post includes another issue the chnage in window which I suyspect has a different cause.

**Comment by slodki** [comment]
Useful links for our CI platform:
[How to code-sign OS X Electron apps in Travis CI](http://jviotti.com/2016/03/16/how-to-code-sign-os-x-electron-apps-in-travis-ci.html)
[Mac: OS X Mavericks (10.9) Code Signing Errors](https://docs.travis-ci.com/user/common-build-problems/#Mac%3A-OS-X-Mavericks-(10.9)-Code-Signing-Errors)

**Comment by wirmike** [comment]
Bestlem,

I apologise for what may seem a tardy reply/response.

I have in the interim had Easter, daughter’s birthday and my wedding anniversary with our daughters sending my wife and me on a break to North Wales.  Lovely - a location that is a close contender to the Med. and Greece!

However, as hinted, this was going to be a learning curve (albeit in reality a small one) for me - a bit like open-heart surgery - on myself!

Anyway - I made myself a ’sudoer’ and then removed the quarantine flag as per the command line below.  And checked the result.

The OSXDAILY information was rather useful as well.

Then I tried to run the MMEXapp from my normal Mac/user and voila!  I worked(!) AND I was in.

I had a look at the last file that I had worked on in the Windows VM and all the changes seemed there.

So thank you for your help.

Being secure can ’seem’ to work against you - but for the greater good.
To quote from the Stackexchange item (70), just prior to the one you suggested, ‘Granted this is a little overkill, but it does the job reliably.’
Caution is the wiser half of a red face!

I look forward to the thoughts on the ideas on ’signing’.

==============================================================================================================================
Just before I sign-off …

In the “Transaction Reports” filter window, at the bottom there is a series of radio-buttons [1,9], just below the text field box.
I use this facility quite often - especially when I have forgotten where I put what in what account - probably a sign of age - but leaving the radio-button on “1”.

Can you say how these are supposed to be used?  I cannot find any guidance in the Documentation and after some experimentation I have not (yet) had any success.
==============================================================================================================================

Thanks again,

MikeD

**Comment by wirmike** [comment]
Bestlem,

Unless you would like me to raise a separate report re the Navigation Pane issue, I will not close this report on Github.

Regards,

Mike

**Comment by wirmike** [comment]
Quarantine issue solved and MMEX now back to normal.

**Comment by slodki** [comment]
No, we have workaround for advanced user only. User should be able to run MMEX without using console commands with admin privileges.

**Comment by wirmike** [comment]
Slodki,

Although the workaround worked, a solution for the ‘basic’ user would be good.
Thanks for re-opening.

Regards,

Mike

**Comment by xraynaud** [comment]
I can confirm the dmg file in the release section works on 10.13. I don't have 10.12 to test.

**Comment by wirmike** [comment]
Slodki,

Well done to the team!

Sounds good!  I am impressed w/your tenaciousness.

Thank you for keeping me ‘in the loop’.

Many thanks,

Wirmike

**Comment by tim-hub** [comment]
it is like app is damaged now.

**Comment by vomikan** [comment]
#2374
#2644 #2459 #2420
